Makefile

```diff
@@ -65,27 +66,27 @@ endif
 #           use source debugging tools like delve.
 all: binary docs
 
-# Build a docker image (skopeobuild) that has everything we need to build.
+# Build a container image (skopeobuild) that has everything we need to build.
 # Then do the build and the output (skopeo) should appear in current dir
 binary: cmd/skopeo
-	docker build ${DOCKER_BUILD_ARGS} -f Dockerfile.build -t skopeobuildimage .
-	docker run --rm --security-opt label:disable -v $$(pwd):/src/github.com/containers/skopeo \
+	${CONTAINER_RUNTIME} build ${CONTAINER_BUILD_ARGS} -f Containerfile.build -t skopeobuildimage .
```

Comment:
`Containerfile.build`? Heh. I can live with that, but this PR needs to actually rename the file or these targets would break.

OTOH there is also a plain `Dockerfile`, used by the `build-container` rule, and renaming that one is a bit more awkward. Sure, not impossible — either way, the two inputs to `$CONTAINER_RUNTIME build` should probably be named consistently.

Reply:
These should still be Dockerfile.  A little too liberal with find/replace

Fixed

Reply:
We can look into changing the Dockerfile afterwards to make sure podman will work inside of the container.